docs/blog/2018-06-26-card-sort-results/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Presenting the Docs Redesign Card Sort Results
+date: 2018-06-26
+author: Shannon Soper
+tags: ["documentation"]
+---
+
+## What is a card sort?
+
+Recently, 36 Gatsby users completed an open card sort to help make [next.gatsbyjs.org](https://next.gatsbyjs.org/) easier to use. Without any outside input or help, they each sorted 90 cards into categories that made sense to them and then named those categories.
+
+### Raw & beautified data
+
+I ran this card sort through [OptimalSort](https://www.optimalworkshop.com/), a software that moderates the card sort and presents the data in usable forms. If you’re curious to see the full results, you can [view the data](https://www.optimalworkshop.com/optimalsort/x87kpp82/5x34psa3-0/shared-results/fa8b66knb66qyhwh5l8j38bd273vkkm7), including raw data as well as things like a similarity matrix and a [dendogram](https://support.optimalworkshop.com/hc/en-us/articles/201997650-Interpreting-the-OptimalSort-dendrograms).
+
+## How did participants categorize the docs?
+
+Most people sorted the docs into the following categories (I included multiple names when the some category names were equally popular):
+* Get Started / Welcome to Gatsby / Intro
+* Development environment
+* Releases & Migration
+* Core Concepts / About Gatsby
+* Advanced Guides
+* API Reference
+* Tutorials / Examples
+* Contributing
+
+These categories weren’t too surprising. However, upon further digging into the data, I discovered some unexpected results that further refine how we might categorize the docs and even write new docs.
+
+## Unexpected results
+
+Some of the results surprised me, and they have significant implications for not just how we organize the docs but also how we think about and talk about Gatsby.
+
+### PWA
+
+No one seems to know what to do with the doc titled “Building apps with Gatsby”. In the similarity matrix data visualization, 41% of people associated it with the “Authentication tutorial” doc and the “Progressive web app” doc (second association pictured below), which was its strongest association with any other doc (and 41% isn’t *very* strong). Essentially, this means participants didn’t think it “fit” very snugly alongside any other docs. 
+
+![Building apps with Gatsby is weakly associated with other docs](building-apps-with-gatsby.png)
+
+This could mean several things:
+1. We don’t have very many docs about how Gatsby can be used to build dynamic apps.
+2. Perhaps people don’t think of Gatsby as a tool they’d use to build apps with
+3. Maybe the title of the doc is too broad or too vague
+4. Another reason I haven’t thought of yet
```

Comment:
you can leave this off, if you want a qualified, "several things, for exmple" works fine.